src/ui/public/notify/__tests__/lib/_format_msg.js

```diff
@@ -26,15 +26,15 @@ describe('formatMsg', function () {
       data: {
         statusCode: 403,
         error: 'Forbidden',
-        message: '[security_exception] action [indices:data/read/mget] is unauthorized for user [user]'
+        message: '[security_exception] action [indices:data/read/msearch] is unauthorized for user [user]'
```

Comment:
These references didn't have to be updated, but I wanted to remove all possible references to mget from client-side code to make future grepping a little easier.